tests/conftest.py

```diff
@@ -1277,12 +1286,39 @@ def pause_cluster_load(request):
     """
     Pause the background cluster load
 
+    """
+    reduce_cluster_load_implementation(request=request, pause=True, resume=False)
+
+
+@pytest.fixture()
+def resume_cluster_load(request):
+    """
+    Pause the background cluster load
+
+    """
+    if config.RUN.get('io_in_bg'):
+
+        def finalizer():
+            """
+            Resume the cluster load
+
+            """
+            resume_cluster_load_implementation()
+
+        request.addfinalizer(finalizer)
+
+
+@pytest.fixture()
+def pause_resume_cluster_load(request):
+    """
+    Pause the background cluster load
+
     """
     reduce_cluster_load_implementation(request=request, pause=True)
 
 
 @pytest.fixture()
-def reduce_cluster_load(request):
+def reduce_resume_cluster_load(request):
     """
     Reduce the background cluster load to be 50% of what it is
```

Comment:
I would add something like: and resume it in the finalizer to original value.

Reply:
added it, please let me know if it's ok now